local_extensions/__init__.py

```diff
@@ -11,19 +12,57 @@
 
 
 VOLTO_MIN_VERSION = 16
-VOLTO_GENERATOR_MIN_VERSION = 6
+
+
+VOLTO_GENERATOR_VERSIONS = {
+    16: (6, 8),
```

Comment:
If I understand correctly. @sneridagh can correct me, if not.

```suggestion
    16: (6, 6),
```

Reply:
The comparison uses `<` rather than `<=` for the max, so I think this should be
```suggestion
    16: (6, 7),
```

Reply:
Ah, you're right. I interpreted it based on its name `maxversion`. That is confusing. I'd either change the comparison to `<=` or its name to `lessthanversion` so that they align with one another.

Reply:
`<=` isn't the semantics we want. We want everything from 6 up to but not including 7.

Reply:
@davisagli is correct.